### PR TITLE
Awesome Book ES6

### PR DESCRIPTION
**Pull request by jeanbuhendwa**
# Awesome Book ES6

### In this pull request, I have : 

- [ ] Created the entry point for your JavaScript code called index.js
- [ ] Created index.css file for styling
- [ ] Created the main index.html file and linked the index.js and index.css 
- [ ] Created a directory called modules and created different modules for different function
- [ ] Imported modules into the index.js file
- [ ] Used let and const to declare variables

### CODE REVIEW

File: modules/books.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    this.bookStore = this.bookStore.filter((book) => book.id !== parentli.id);
+    bookList.removeChild(e.target.parentElement);
+    this.updateLocalStorage();
+  }
+
+  restoreLocalStorage() {
+    this.bookStore = JSON.parse(localStorage.getItem('book_list'));
+    this.bookCount = JSON.parse(localStorage.getItem('book_count'));
+    for (let i = 0; i < this.bookStore.length; i += 1) {
+      this.displayBook(this.bookStore[i]);
```
Comment (Munsa1) [inline modules/books.js]: - [ ] Kindly use arrow functions for these functions as this is the point of thhhis milestone.

File: modules/books.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    }
+  }
+
+  updateLocalStorage() {
+    localStorage.setItem('book_list', JSON.stringify(this.bookStore));
+    localStorage.setItem('book_count', JSON.stringify(this.bookCount));
+  }
+}
```
Comment (Munsa1) [inline modules/books.js]: - [ ] Again here it would be nice to use an arrow function as this is a requirement.

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @jeanbuhendwa, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
Added time using `luxon` :heavy_check_mark: 
Correct GitHub Flow :heavy_check_mark: 
Descriptive ReadMe File :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by thecodechaser — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @jeanbuhendwa👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- Highly organized code. ✔️
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**